### PR TITLE
Add lint rule "no-restricted-imports" for lodash and date-fns 

### PR DESCRIPTION
**Pull request by arnifreyrs**
# \<Reduce bundle size due to importing lodash and date-fns\>

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

**Review by darrikonn — APPROVED** [review]
AWESOME!

**Review by jeremybarbet — APPROVED** [review]
LGTM

**Review by gudjong — APPROVED** [review]
Nice

**Review by gudjong — CHANGES_REQUESTED** [review]
I approved this a bit quickly. There is a failing test in the judicial system backend. I'll can investigate if you like.

**Comment by arnifreyrs** [comment]
I am looking into it @gudjong

**Review by Thorrik58 — APPROVED** [review]
Looks good to me!

**Review by jeremybarbet — APPROVED** [review]
LGTM

**Review by saevarma — CHANGES_REQUESTED** [review]

File: libs/judicial-system/formatters/src/lib/formatters.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
-import { isValid, format, parseISO } from 'date-fns'
-import { is } from 'date-fns/locale'
+// eslint-disable-next-line no-restricted-imports
+import { format, parseISO, isValid } from 'date-fns'
```
[the remark this answers is not in the document]
Comment (arnifreyrs) [reply libs/judicial-system/formatters/src/lib/formatters.ts:2]: I tried to have this the new way for 2 hours, but doing that introduced some test failures that required immense changes to tsconfig.json for this lib and the app that required it so I just gave up after that long. I feel like this should just slide and be fixed if wished by the owners of the judicial system.

[the remark this answers is not in the document]
Comment (saevarma) [reply libs/judicial-system/formatters/src/lib/formatters.ts:2]: Ok sounds valid.

**Review by saevarma — APPROVED** [review]

**Review by darrikonn — APPROVED** [review]

**Pull request merged**